### PR TITLE
fix(test): Add continuous integration using Github actions

### PR DESCRIPTION
**Pull request by ERosendo**
This PR solves multiple issues related to the implementation of continuous integration.

- Remove references to Travis CI ( in `readme.md` and `testing.md`)  and also delete the configuration file ( travis.yml).
- Adds [Puppeteer](https://pptr.dev/) as a dev dependency to use its Chromium binary to run the tests in [headless](https://developers.google.com/web/updates/2017/04/headless-chrome) mode.
- Update the configuration file used by Karma to change the default browser used to run the tests. 
- Bump Karma from 6.3.2 to 6.4.1 to avoid getting  `DeprecationWarning` while running the test suite.

This changes should close [Fix continuous integration#310](https://github.com/freelawproject/recap/issues/310).

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+## Current
+
+**1.2.30 - 2022-11-09**
+
+Changes:
+ - Remove references to Travis CI in readme.md and testing.md files.
+ - Delete the configuration file for Travis CI.
+ - Add the test.yml file to define a workflow that runs tests using GitHub Actions.
+ - Adds [Puppeteer](https://pptr.dev/) as a dev dependency to use its Chromium binary to run the tests in [headless](https://developers.google.com/web/updates/2017/04/headless-chrome) mode.
+ - Update the configuration file used by Karma to change the default browser used to run the tests.
```
Comment (mlissner) [inline CHANGES.md:25]: We don't want to bump the version yet. We'll do that when we are ready to cut a release. So:

```suggestion
# Change Log

## Upcoming

The following changes are not yet released, but are code complete:

Features:
 - None
 
Changes:
- None

Fixes:
- More robust uploading of attachment pages ([#304](https://github.com/freelawproject/recap/issues/304), [#238](https://github.com/freelawproject/recap/issues/238), [#291](https://github.com/freelawproject/recap/issues/291))

For Developers:
 - Migrates from Travis CI to Github Actions
 - Adds [Puppeteer](https://pptr.dev/) as a dev dependency to use its Chromium binary to run the tests in [headless](https://developers.google.com/web/updates/2017/04/headless-chrome) mode.
 - Updates the configuration file used by Karma to change the default browser used to run the tests.

## Current & Previous

See releases: https://github.com/freelawproject/recap-chrome/releases/
```

File: package.json [hunk 2]
```diff
@@ -1,13 +1,13 @@
 {
   "name": "recap-chrome",
-  "version": "1.2.29",
+  "version": "1.2.30",
```
Comment (mlissner) [inline package.json:3]: Let's not do this yet, please.

**Review by mlissner — CHANGES_REQUESTED** [review]
Looks great. A couple little tweaks, please.

File: package.json (hunk 2 above)
Comment (ERosendo) [reply package.json]: Changed. Thanks!

**Comment by mlissner** [comment]
Great!

**Pull request merged**